src/sidebar/store/modules/test/groups-test.js

```diff
@@ -71,7 +71,9 @@ describe('sidebar/store/modules/groups', () => {
   let store;
 
   beforeEach(() => {
-    store = createStore([groups, session]);
+    // The empty second argument (settings) needed here because of the
+    // dependency on the `session` module
+    store = createStore([groups, session], [{}]);
```

Comment:
Perhaps give `settings` a default value of `{}` in the `init` function? I'm not sure if that will work given the `createStore` implementation.